controllers/profiles/reconciler_dev.go

```diff
@@ -120,9 +127,30 @@ func (e *ensureRunningDevWorkflowReconciliationState) Do(ctx context.Context, wo
 	}
 	objs = append(objs, propsCM)
 
+	devBaseContainerImage := defaultKogitoServerlessWorkflowDevImage + ":" + version.OperatorVersion
+	pl, errPl := platform.GetActivePlatform(ctx, e.client, workflow.Namespace)
+	// check if the Platform available
+	if errPl == nil {
+
+		// if is present the optional devBaseImage
+		if len(pl.Spec.DevBaseImage) > 0 {
+			devBaseContainerImage = pl.Spec.DevBaseImage
+		} else {
+			// If is not present the optional devBaseImage and is present the baseImage
+			if len(pl.Spec.BuildPlatform.BaseImage) > 0 {
+				devBaseContainerImage = pl.Spec.BuildPlatform.BaseImage
+			}
+		}
+	} else {
+		// is the operator version is a snapshot we add the nightly version
+		if isSnapshot(version.OperatorVersion) {
+			devBaseContainerImage = defaultKogitoServerlessWorkflowDevImage + "-" + nightlySuffix + ":latest"
```

Comment:
```suggestion
			devBaseContainerImage = defaultKogitoServerlessWorkflowDevImage + "-" + nightlySuffix + ":latest"
```

For the dev image version, we should be using:
- `latest` if version is `2.0.0-snapshot`
- `X.Y` if version is another snapshot version (for example 1.34, 1.35). See https://quay.io/repository/kiegroup/kogito-swf-builder-nightly?tab=tags

Reply:
after discussion, and the graph on the issue, let's keep it like that.
Our tests should set the correct devbaseimage if needed